src/utils/__tests__/api.test.ts

```diff
@@ -137,7 +137,7 @@ describe("Bancomat Pay: getting the list", () => {
     });
     global.fetch = jest.fn(() => Promise.resolve(response));
     const result = await pm.bPay.getList(sessionToken);
-    expect(result).toEqual(E.left(ErrorsType.GENERIC_ERROR));
+    expect(result).toEqual(E.right([]));
```

Comment:
Please consider to update the test's name. Something like this should be ok: "should return an empty array when http === 200 but no bpay item"

Reply:
You're right, fixing